stream_alert/alert_processor/outputs.py

```diff
@@ -37,6 +37,8 @@
 # All included subclasses are designated using the '@output' class decorator
 # The keys are the name of the service and the value is the class itself
 # {cls.__service__: <cls>}
+
+# pylint: disable=too-many-lines
```

Comment:
Note to other reviewers: I asked @armtash to add this for now with the intent that I would break this out after this PR merges.